Automattic/wp-calypso
# DIFM content form: dialog updates & feedback char limit

Fixes https://github.com/Automattic/martech/issues/3432

## Proposed Changes

* Updates copy for the "content submissions guidelines" dialog and the "confirm submission" dialog.
* Adds a character limit for the Feedback text box.

## Why are these changes being made?


* pdh1Xd-2Ue-p2

## Testing Instructions



* Go to `/start/do-it-for-me` and purchase the product.
* After checkout, once on the content submission form, confirm that you see this dialog and confirm that the copy matches the one on https://wordpress.com/support/website-design-service/#content-guidelines
  <img width="907" alt="image" src="https://github.com/user-attachments/assets/f69ff1e5-3699-4b1c-bca7-8a8245164510">
* Go to the "Submit Content" section and confirm that the text box has a character limit of 1000 characters and does not accept any input after 100 characters.
  <img width="853" alt="Screenshot 2024-10-22 at 4 24 35 PM" src="https://github.com/user-attachments/assets/39df6b1f-853f-48e0-b678-6d4513b5cf74">
* Click on the "Submit" button and confirm that the dialog copy is the same as the following:
 <img width="823" alt="Screenshot 2024-10-22 at 4 24 39 PM" src="https://github.com/user-attachments/assets/8de70723-78d2-4520-9213-92939a498415">



## Pre-merge Checklist
